tests/e2e/utils/process.ts

```diff
@@ -27,6 +27,15 @@ function _exec(options: ExecOptions, cmd: string, args: string[]): Promise<Proce
     `==========================================================================================`
   );
 
+  // Auto-add some flags to ng commands that build or test the app.
+  // --no-progress disables progress logging, which in CI logs thousands of lines.
+  // --no-sourcemaps disables sourcemaps, making builds faster.
+  // We add these flags before other args so that they can be overriden.
+  // e.g. `--no-sourcemaps --sourcemaps` will still generate sourcemaps.
+  if (cmd === 'ng' && ['build', 'serve', 'test', 'e2e', 'xi18n'].indexOf(args[0]) !== -1) {
+    args = [args[0], '--no-progress', '--no-sourcemaps', ...args.slice(1)]
```

Comment:
Feels like we should have default values for those in the config, but I cannot find them. Ideas?